projects/example-app/src/app/reducers/index.ts

```diff
@@ -60,10 +58,15 @@ export function logger(reducer: ActionReducer<State>): ActionReducer<State> {
  * By default, @ngrx/store uses combineReducers with the reducer map to compose
  * the root meta-reducer. To add more meta-reducers, provide an array of meta-reducers
  * that will be composed to form the root meta-reducer.
+ *
+ * export const metaReducers: MetaReducer<State>[] = !environment.production
+ *  ? [logger]
+ *  : [];
+ *
+ * Since V15.1 Angular doesn't include environment files by default.
+ * If you want to use meta-reducers please add it manually using
+ * `ng g environments` command.
  */
-export const metaReducers: MetaReducer<State>[] = !environment.production
-  ? [logger]
-  : [];
```

Comment:
We can use `isDevMode` instead of `!environment.production` here:

```ts
import { isDevMode } from '@angular/core';

export const metaReducers: MetaReducer<State>[] = isDevMode()
  ? [logger]
  : [];
```

`environment.production` could also be replaced with `!isDevMode()` here: https://github.com/ngrx/platform/blob/master/projects/example-app/src/app/app.module.ts#L67

Reply:
@markostanimirovic 
Implemented in [604d438](https://github.com/ngrx/platform/pull/3735/commits/604d43893725256eeca046b1966d6afe019dcfe0)